components/webui/imports/utils/datetime.js

```diff
@@ -3,12 +3,14 @@ import Duration from "dayjs/plugin/duration";
 import Timezone from "dayjs/plugin/timezone";
 import Utc from "dayjs/plugin/utc";
 
+
 dayjs.extend(Utc);
 dayjs.extend(Timezone);
 dayjs.extend(Duration);
 
 const DATETIME_FORMAT_TEMPLATE = "YYYY-MMM-DD HH:mm:ss";
 
+/* eslint-disable sort-keys */
 const TIME_UNIT = Object.freeze({
```

Comment:
We should add descriptions for all Enums. To automate this, we can add an element into the ESLint rule `jsdoc/require-jsdoc`'s `contexts` as follows:
``` json
{
  "jsdoc/require-jsdoc": [
    "warn",
    {
      "contexts": [
        ":has(VariableDeclarator[init.callee.object.name='Object'][init.callee.property.name='freeze'])[declarations]"
      ],
      "require": {
        "ArrowFunctionExpression": true,
        "FunctionDeclaration": true,
        "FunctionExpression": true
      }
    }
  ]
}
```

(Took me quite a while to figure out how to select only the `const` part rather than the VariableDeclarator (e.g. `TIME_UNIT`), or the rule checker would not detect the valid JSDoc above the code; even worse, the rule fixer would insert dummy comment blocks between `const` and the VariableDeclarator. phew..

Ref:
1. https://eslint.org/docs/latest/extend/selectors
2. https://estools.github.io/esquery/
3. https://astexplorer.net/  (JavaScript: @babel/parser)

Reply:
Let's also configure `"publicOnly": true` to avoid the handler functions within React components getting reported. For those functions, if they have any parameters, any undocumented function would be reported by rule [`require-param`](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/require-param.md).

Reply:
> Let's also configure "publicOnly": true

Probably not a good idea.. With that setting, any non-exported Enums that are undocumented would be get reported. Let me see if there's a way to enable `ArrowFunctionExpression` without reporting the ones in React elements.